Assets/MixedRealityToolkit.Services/InputAnimation/InputRecordingService.cs

```diff
@@ -270,7 +270,7 @@ private bool RecordInputHandData(Handedness handedness)
         /// <inheritdoc />
```

Comment:
I tried pulling this branch and cannot seem to get a record working. In particular I get a nullreferenceexception for OnRecordingStarted & OnRecordingStopped. These should both be accessed via OnRecordingStart?.Invoke();

Also when I hit record/stop in the HandInteractionRecordArticulatedHands example scene, I don't see any file posted? 

Also the materials in the HandInteractionRecordArticulatedHands example scene use the Unity standard shaders

Reply:
Probably the input service data providers got messed up again, it happens all the time because MRTK doesn't have proper versioning ...

Reply:
You should be able to fix it manually by setting the data provider types in the input system profile for the recording and playback services. I'll have a look after merging mrtk_dev.